tests/test_launcher.py

```
@@ -82,95 +84,86 @@ def test_incompatible_version(self):
 class TestVirtualEnvironment:
     """Test virtual environment creation and management."""
 
-    @patch("launch.venv.create")
-    @patch("launch.Path.exists", return_value=False)
+    @patch("setup.environment.venv.create")
+    @patch("setup.environment.Path.exists", return_value=False)
     def test_create_venv_success(self, mock_exists, mock_venv_create):
         """Test successful venv creation."""
         venv_path = ROOT_DIR / "venv"
-        with patch("launch.logger") as mock_logger:
+        with patch("setup.environment.logger") as mock_logger:
```

Flake8 (7.3.0)

[error] local variable 'mock_logger' is assigned to but never used

(F841)